frontend/javascripts/oxalis/model/bucket_data_handling/bucket_picker_strategies/flight_bucket_picker.js

```diff
@@ -67,7 +67,7 @@ export default function determineBucketsForFlight(
   const cameraDirection = V3.sub(centerPosition, cameraPosition);
   V3.scale(cameraDirection, 1 / Math.abs(V3.length(cameraDirection)), cameraDirection);
 
-  const iterStep = 10;
+  const iterStep = 8;
```

Comment:
I also tweaked the values for the arbitrary mode pickers a bit, since they are not that perf criticial anymore.